Sources/TuistLoaderTesting/Utils/Mocks/MockSetupLocator.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import TSCBasic
+@testable import TuistLoader
+
+public final class MockSetupLocator: SetupLocating {
+    var invokedLocate = false
```

Comment:
- some of these properties are not used
- I'd prefer using a more light-weight stubbing with `var locate: ((AbsolutePath) -> AbsolutePath?)?` unless you need to reuse some stubbing functionality in a lot of places